include/dem/distributions.h

```diff
@@ -31,20 +31,24 @@ class Distribution
   /**
    * @brief Carries out the size sampling of particles. This is the base class of
    * NormalDistribution, UniformDistribution, LogNormalDistribution and
-   * HistogramDistribution classes.
+   * CustomDistribution classes.
```

Comment:
```suggestion
```

Reply:
You mean this? 

```suggestion
   NormalDistribution, UniformDistribution  and CustomDistribution classes.
```

Reply:
I should remove LogNormalDistribution, not CustomDistribution...